backends-velox/src/main/scala/org/apache/spark/sql/execution/datasources/velox/VeloxFormatWriterInjects.scala

```diff
@@ -45,8 +45,20 @@ trait VeloxFormatWriterInjects extends GlutenFormatWriterInjectsBase {
       dataSchema: StructType,
       context: TaskAttemptContext,
       nativeConf: java.util.Map[String, String]): OutputWriter = {
+    // The path parameter is same with the method name of OutputWriter.
```

Comment:
@PHILO-HE Add comments for why need to rename the path parameter to originPath.

Reply:
Thanks for your feedback! Can we just use a no conflict name for argument in `createOutputWriter(path: String..`?

Reply:
@PHILO-HE Updated. Thanks for your suggestions.